department_management/src/main.rs

```diff
@@ -1,11 +1,17 @@
+//! Exercising in managing department.
+//!
+//! # Task
+//!
+//! Using a hash map and vectors, create a text interface to allow a user to add
+//! employee names to a department in a company. For example, `Add Sally to
+//! Engineering` or `Add Amir to Sales`. Then let the user retrieve a list of
+//! all people in a department or all people in the company by department,
+//! sorted alphabetically.
+
 use std;
 use std::collections::HashMap;
```

Comment:
You don't need type `use std;`  ever. Just use `std::` stuff directly and it will work. This works also with any crate, not just `std`, as you're using full path here. Full path doesn't require root segment to be `use`d.

Examples: `log::info!()`, `std::fmt::Debug`, `serde_json::from_str()`.